### PR TITLE
docs: update teams information

### PR DESCRIPTION
**Pull request by tancnle**
# Description

Update teams information

- Remove departed team members
- Consolidate SplashKit extension team links (MS Teams channel and Trello board)
- Fix a small typo in the channel name

## Type of change

- [x] Documentation (update or new)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: docs/projects/t1-2022/teams.md
Comment (tancnle) [inline docs/projects/t1-2022/teams.md:126]: The two boards have been consolidated into one. Could you confirm this @allanfarrell? 🙏🏼

**Review by ghost — APPROVED** [review]
Looks good, ready to merge once approved by Allan and Alex!
(Avoiding tags as they are already mentioned)

**Comment by tancnle** [comment]
@Alex-MobileApps @allanfarrell Slight nudge. Can you kindly review and approve if it looks ok 🙏🏼 ?

**Comment by ghost** [comment]
Confirmed with Allan in MS Teams call 7/4/22

**Comment by tancnle** [comment]
Thank you for following up @talzid 🙇🏼

**Pull request merged**
